File: back-end/ConstructionFinance/src/main/java/br/com/galsystem/construction/finance/dto/workday/WorkDayBulkPaymentDTO.java
package br.com.galsystem.construction.finance.dto.workday;

import java.time.LocalDate;
import java.util.List;

public record WorkDayBulkPaymentDTO(
        List<Long> workdayIds,
        Long supplierId,
        String description,
        LocalDate paymentDate,
        Long payerId,
        Long categoryId,
        Long serviceContractId,
        String paymentMethod,
        Double amount
) {
}

File: front-end/src/app/pages/workday/WorkdayBulkPaymentDTO.ts
export interface WorkdayBulkPaymentDTO {
  workdayIds: number[];
  supplierId: number;
  description: string;
  paymentDate: string;
  payerId: number;
  categoryId: number;
  serviceContractId: number;
  paymentMethod: string;
  amount: number;
}
